apps/sms/js/link_helper.js

```diff
@@ -58,7 +58,7 @@ function checkDomain(domain) {
 }
 
 // defines things that can match right before to be a "safe" link
-var safeStart = /[\s,:;\(>]/;
+var safeStart = /[\s,:;\(>\u0080-\uFFFF]/;
```

Comment:
I think it depends on app, and you could see each app have their different logic for handling the link :)